.travis.yml

```diff
@@ -0,0 +1,12 @@
+branches:
+  except:
+    - master
+language: ruby
+cache: bundler
+sudo: false
+rvm:
+- 2.2.3
+script:
+- bundle exec rubocop
+- bundle exec foodcritic -f any .
+- bundle exec rspec test/unit
```

Comment:
Is there a way to run manual travis checks with different configs? I'd like to play with it a bit to try and get a chef-zero run and run inspec, if not kitchen as a whole (there's no way to run vagrant boxes on travis, so we would end up running chef locally and testing that either way)